src/SmartComponents/BaselinesPage/EditBaseline/helpers.js

```diff
@@ -140,6 +140,42 @@ function makeDeleteSubFactPatch(factToDelete, parentFact, originalAPIBody) {
     return newAPIBody;
 }
 
+function makeDeleteFactsPatch(factsToDelete, originalAPIBody) {
+    let patchBody = [];
+
+    factsToDelete.forEach(function(fact, index) {
+        if (!fact.selected) {
+            if (Array.isArray(fact[2])) {
```

Comment:
What does `fact[2] represent? Casting to a variable with a name that identifies what it is would help here.